src/ui/components/buttons/RoundedButton.tsx

```diff
@@ -34,6 +36,8 @@ interface Props {
 
 const getIcon = (iconName: Props['iconName']): React.FC<AccessibleIcon> => {
   if (iconName === 'back') return ArrowPrevious
+  if (iconName === 'next') return ArrowRightNew
+  if (iconName === 'previous') return ArrowLeftNew
```

Comment:
Pourquoi a-t-on besoin de suffixer `New` ?

```suggestion
  if (iconName === 'next') return ArrowRight
  if (iconName === 'previous') return ArrowLeft
```

Reply:
`ArrowRight`et `ArrowLeft` existent déjà (ce sont des icônes par rapport à la refonte).
A terme il ne vont pas être gardés pour être remplacés par les `New` qu'on renommera à ce moment là. 
J'ai vu avec Aline